the-experts-BE/README.md

```diff
@@ -0,0 +1,55 @@
+##### the-experts
+
+## [ OUR HOSTED Experts {Not Working For NOW} ](https://the-experts.herokuapp.com/)
+
+## Local installation
+```
+ git clone https://github.com/FACN7/the-experts.git
+ cd the-experts
+ npm i 
+```
+In the project directory AND in the the-experts-fe directory, you have to run:
+
+### `npm start`
+
+Runs the app in the development mode.<br />
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.<br />
+You will also see any lint errors in the console.
+
+## User journey
+Well, it's and app for connecting contractors with users, you can enter to the app and search for a contractor
+who can help you to fix your home issues.
+
+
+## dev journal
+For this project we had to:<br />
+* Use React as FE & express as BE
+* Fetch contractor's data from our DB
+* Display our data
+* host to heroku
+## What we did:
+
+- [x] created a react app
+- [x] created an express app
+- [ ] fetch contractor's data from our DB
+- [ ] display data
+- [ ] host to heroku
```

Comment:
You've now done `fetch contractor's data from our DB` / `display data` / `host to heroku`. These can be ticked off 👍